src/app/landing-page/login/login.component.html

```diff
@@ -58,12 +61,13 @@ <h2>Anmeldung</h2>
             }
           </div>
       </div>
+
       <div class="btn-container">
         <button class="login-btn" type="submit">Anmelden</button>
         <button class="guest-btn" routerLink="main">Gäste-Login</button>
       </div>
       <!-- pwd recovery link -->
-      <button class="btn-password fontNunito" routerLink= '/recovery' title="Passwort vergessen?" aria-label="Link zur Passwort wiederherstellung">Passwort vergessen?</button>
+      <button class="btn-password" routerLink= '/recovery' title="Passwort vergessen?" aria-label="Link zur Passwort wiederherstellung">Passwort vergessen?</button>
```

Comment:
Obsolete `title` attribute. If it links to another page, it should instead be a `<a>` tag instead of `<button>`. If you change the button to a, you don't need the `aria-label` attribute.